src/FSharpLint.Core/Text.resx

```diff
@@ -368,5 +368,8 @@
   </data>
   <data name="RulesEnsureTailCallDiagnosticsInRecursiveFunctions" xml:space="preserve">
     <value>The '{0}' function has a "rec" keyword, but no [&lt;TailCall&gt;] attribute. Consider adding [&lt;TailCall&gt;] attribute to the function and &lt;WarningsAsErrors&gt;FS3569&lt;/WarningsAsErrors&gt; property to project file (but only on .NET 8 and higher).</value>
-  </data>  
+  </data>
+  <data name="RulesFavourAsKeyword" xml:space="preserve">
+    <value>Prefer using the as pattern to match a constant and bind it to a variable.</value>
```

Comment:
@ribeirotomas1904 nit, let's wrap the word "as" with single-quotes, to make this more readable (otherwise at first glance it's hard to see that `as` is meant to be read as a keyword)

Reply:
done in c985a3162898b5c13fd73ac2c71ec465df3538e3